### DISCUSSION TITLE
Include recipe in other recipe

### DISCUSSION THREAD

**Discussion by vincentDcmps** [mealie-recipes/mealie#3536] [Ideas]
Hi everyone,

I just thought to a new feature, maybe it will be interesting to be able to include recipe with ingredient list in another recipe.
example for sausage recipe, that you include in a meal recipe. this avoid to enter the sausage recipe in each meal recipe.

**Comment by boc-the-git** [comment]
Please include any thoughts on the existing discussion: https://github.com/mealie-recipes/mealie/discussions/1761
